tests/system/Helpers/InflectorHelperTest.php

```diff
@@ -244,12 +244,6 @@ public function testDasherize()
     public function testOrdinal()
     {
         $suffixes = [
-            'st' => 1,
```

Comment:
This test should be revised to use data provider instead of deleting the test values

Reply:
implemented.